src/index.js

```diff
@@ -152,6 +152,34 @@ module.exports = function(source, inputSourceMap) {
     options.sourceFileName = relative(options.sourceRoot, options.filename);
   }
 
+  if (this.version > 1 && options.presets) {
+    const presetsWithModulesOption = ["es2015", "env", "latest"];
```

Comment:
```js
const presets = [ "es2015", "env", "latest" ]

// not sure if really needed
const name = (name) => name
  .replace(/^@babel\//, "")
  .replace(/^preset-/, "")

options.presets.forEach((preset) => presets.indexOf(name(preset[0])) > -1 
   ? preset[1] !== undefined 
     ? preset[1].modules = false
     : preset[1] = { modules: false }
   : preset
)

Reply:
But I agree with @danez [here](https://github.com/babel/babel-loader/pull/529#issuecomment-342196655) that a warning with link(s) to the appropriate docs would be the saner solution. `babel` is configurable and modular by design and a dev simply needs to know how to configure it properly